ai-player.js

```diff
@@ -525,28 +528,28 @@ function createAiPlayer(game, options) {
         return roles;
     }
 
-    function revealLowestRanked() {
+    function revealByProbability() {
         var influence = ourInfluence();
-        var i = rankedRoles.length;
-        while (--i) {
-            var role = rankedRoles[i];
-            if (influence.indexOf(role) >= 0) {
-                command({
-                    command: 'reveal',
-                    role: role
-                });
-                // Don't claim this role any more.
-                if (claims[state.playerIdx]) {
-                    delete claims[state.playerIdx][role];
+        var chosenInfluence = 0;
+
+        if (influence.length > 1) {
+            var influenceProbability = [];
+            for(var i = 0; i < influence.length; i++) {
```

Comment:
Nit: the convention used elsewhere in the source is for a space between `for` and `(`